intel_extension_for_transformers/neural_chat/examples/talkingbot/front_end/README.md

```diff
@@ -0,0 +1,31 @@
+<h1 align="center" id="title"><img align="center" src="./src/lib/assets/favicon.png" alt="project-image">Talking Bot</h1>
+
+<h2>🚀 Demo</h2>
+
+[https://neuralstudio.eglb.intel.com](https://neuralstudio.eglb.intel.com)
```

Comment:
this is not approved, better to use localhost now?

Reply:
Because there is no public link, here is a screenshot